backend/src/main/java/ai/verta/modeldb/versioning/blob/container/DatasetContainer.java

```diff
@@ -78,6 +83,19 @@ public void process(
         throw new ModelDBException("Unknown blob type", Code.INTERNAL);
     }
     if (blobHash != null) {
+      Blob blob = super.getBlob();
+      try {
+        List<AttributeEntity> attributeEntities =
+            RdbmsUtils.convertAttributesFromAttributeEntityList(
+                this, ModelDBConstants.ATTRIBUTES, blob.getAttributesList());
+        attributeEntities.forEach(
+            attributeEntity -> {
+              attributeEntity.setEntity_hash(blobHash);
```

Comment:
this will need to be a concatenation of repoid, commit_hash and location.

Reply:
PR: https://github.com/VertaAI/modeldb/pull/774